src/main/java/com/firstspringmvcapp/controllers/ApiController.java

```diff
@@ -0,0 +1,97 @@
+package com.firstspringmvcapp.controllers;
+
+import com.firstspringmvcapp.util.PersonNotFoundException;
+import jakarta.validation.Valid;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.annotation.*;
+import com.firstspringmvcapp.dto.PersonDTO;
+import com.firstspringmvcapp.models.Person;
+import com.firstspringmvcapp.security.JWTUtil;
+import com.firstspringmvcapp.services.PeopleService;
+import com.firstspringmvcapp.util.PersonErrorResponse;
+import com.firstspringmvcapp.util.PersonNotCreatedException;
+import com.firstspringmvcapp.util.PersonValidator;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
```

Comment:
It's a super abstract name and path
This controller has 1 responsibility - give resources to manage users 
So lets rename it to UserController  with path /users
And you can add the version of api to path (v1/users for example) refer to https://restfulapi.net/resource-naming/